### ISSUE TITLE
.exe or .msi installer (not everyone wants win10)

### ISSUE THREAD

**Issue by atesin** [VladimirAkopyan/Authenticator#13]
as many people i guess, i don't like windows 10 and won't change just for use an app,

please... make this app compatible with WINDOWS (any) by enabling this

**Comment by LoganDark** [comment]
UWP is Windows 10 only, so even if this was not an appx, it wouldn't even run on older versions.

**Comment by atesin** [comment]
yes i know uwp is exclusive for win10, that's what i'm talking about ... i addition to package your application in UWP do it in .msi .exe or .zip also
